### PR TITLE
Today button

### PR DESCRIPTION
**Pull request by martijnrusschen**
Add today button and the example.

### CODE REVIEW

File: src/calendar.jsx [hunk 1]
```diff
@@ -2,6 +2,15 @@ import YearDropdown from "./year_dropdown";
 import Month from "./month";
 import React from "react";
 
+var todayTr = {
```
Comment (rafeememon) [inline src/calendar.jsx]: This is kind of yucky.  Maybe we can allow the today button text to be passed in as a prop, or can we get it out of localeData?

File: src/datepicker.jsx [hunk 2]
```diff
@@ -36,7 +37,9 @@ var DatePicker = React.createClass({
       onChange() {},
       disabled: false,
       onFocus() {},
-      onBlur() {}
+      onBlur() {},
+      onClear() {},
```
Comment (rafeememon) [inline src/datepicker.jsx]: Looks like onClear snuck back in here

File: src/datepicker.jsx [hunk 3]
```diff
@@ -36,7 +37,9 @@ var DatePicker = React.createClass({
...
+      showTodayButton: false
```
Comment (rafeememon) [inline src/datepicker.jsx]: Probably don't need this default, falsy would be implied by not passing in the prop

Comment (martijnrusschen) [inline src/datepicker.jsx]: Correct.

File: src/datepicker.jsx (hunk 2 above)
Comment (martijnrusschen) [inline src/datepicker.jsx]: Fixed

File: src/calendar.jsx (hunk 1 above)
Comment (martijnrusschen) [inline src/calendar.jsx]: Moved this to a prop.

**Comment by martijnrusschen** [comment]
Added some tests for the today button. Still need to add tests to test the actual onClick

**Comment by martijnrusschen** [comment]
Looks like I screwed up a merge conflict..

**Comment by martijnrusschen** [comment]
@rafeememon Can you help out here? Not sure what the should be removed.

**Pull request closed without merging**